sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -427,6 +436,21 @@ define( [
                                 Notify.warn( 'detected differents target instance with the same name (' + name + ')' );
                         }
                     }
+                } else if ( animationCallback instanceof UpdateMorph ) {
```

Comment:
adds some empty line in the code check the coding style https://github.com/cedricpinson/osgjs/wiki/Coding-Style